web/wp-content/themes/borealis-2021/inc/custom-post-types/news-cpt.php

```diff
@@ -0,0 +1,49 @@
+<?php 
+
+if (!function_exists('pg_register_news_cpt')) {
+    /**
+     * Creates a Location Custom Post Type with a services custom taxonomy and custom metadata
+     */
+    function pg_register_news_cpt() {
+        // Arguments are:
+        // - slug.
+        // - single.
+        // - plural.
+        // - args (an optional array to overwrite any default settings).
+        $namespace = pg_get_namespace();
+        $News_CPT = new PG_Custom_Post_Type('news', 'News', 'News', array('icon' => 'dashicons-category', 'has_archive' => 'false', 'template' => [ [$namespace . '/news-meta-block']]));
+        $News_CPT->register();
+        $News_CPT->register_tags('research-area', 'Research-area', 'Research-areas');
+
+        // Register Meta.
+        // Slug will automatically be pulled from when it's registered.
+        $meta_values = array(
+            'publication_date' => 'text',
+            'description' => 'text',
+            'source_publication' => 'text',
+            'authors' => 'text',
+            'external_link' => 'text',
+            'image_alt' => 'text',
+            'image_url' => 'text',
+            'image_id' => 'number',
```

Comment:
Similarly please remove the 'image_' meta values and the 'description', we'll use some WP functionality for those